### PR TITLE
Fix missing error description for OpenSSL protocol error

### PR DESCRIPTION
**Pull request by pepone**
Fix for #1391

### CODE REVIEW

**Review by externl — APPROVED** [review]
Looks good!

File: CHANGELOG-3.7.md [hunk 1]
```diff
@@ -72,15 +74,26 @@ particular aspect of Ice.
...
+
+# Changes in Ice 3.7.9
+
+These are the changes since Ice 3.7.8.
+
+## C++ Changes
+
+- Fixed a bug in the OpenSSL implementation of IceSSL resulting in the SSL protocol error description not being
+  included with the `Ice::ProtocolException` exception that is thrown when an SSL protocol error is detected during
+  reading. OpenSSL version previous to `1.1.1e` are not affected.
```
Comment (InsertCreativityHere) [inline CHANGELOG-3.7.md:95]: Everything else looks perfect!
```suggestion
  reading. OpenSSL versions previous to `1.1.1e` are not affected.
```

**Review by InsertCreativityHere — APPROVED** [review]

File: CHANGELOG-3.7.md [hunk 2]
```diff
@@ -72,15 +74,26 @@ particular aspect of Ice.
...
+
+# Changes in Ice 3.7.9
+
+These are the changes since Ice 3.7.8.
+
+## C++ Changes
+
+- Fixed a bug in the OpenSSL implementation of IceSSL resulting in the SSL protocol error description not being
+  included with the `Ice::ProtocolException` exception that is thrown when an SSL protocol error is detected during
+  reading. OpenSSL versions previous to `1.1.1e` are not affected.
```
Comment (bentoi) [inline CHANGELOG-3.7.md:95]: The sentence is a bit too long 🙂. Perhaps: 

Fixed an IceSSL bug where the OpenSSL error description was not correctly with the `Ice::ProtocolException` when a read failure occurred.

**Review by bentoi — APPROVED** [review]
Looks good.

**Pull request merged**